docs/tutorial/force_calibration.rst

```diff
@@ -0,0 +1,95 @@
+Force calibration
+=================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+Force calibration is used to convert the raw voltages that Bluelake records to actual forces.
+In this tutorial, we will be redoing a calibration that we already performed in Bluelake, to provide an example of using the force calibration module::
+
+    f = lk.File("calibration.h5")
+
+In this tutorial, we want to reproduce calibration item 1.
+We grab the chunk of data that was used for the calibration last time::
+
+    start = f.force1x.calibration[1]["Start time (ns)"]
+    stop = f.force1x.calibration[1]["Stop time (ns)"]
+    force_slice = f.force1x[start:stop]
+
+To be able to calibrate our forces, we first convert back our data to raw voltages.
+We can do this using the previous calibration performed in Bluelake (in this case, the first one in the file)::
+
+    volts = force_slice.data / f.force1x.calibration[0]["Rf (pN/V)"]
+
+Force calibration models are fit to power spectra. To compute a power spectrum from our data::
+
+    power_spectrum = lk.calculate_power_spectrum(volts, sample_rate=force_slice.sample_rate)
+
+This function returns a power_spectrum which we can plot::
+
+    power_spectrum.plot()
+
+.. image:: force_calibration_blocked_spectrum.png
+
+Note that the computation of the power spectrum involves some downsampling.
+If you wish to change the amount of downsampling applied or which range to compute the spectrum for, you can specify these as additional arguments::
+
+    lk.calculate_power_spectrum(volts, sample_rate=force_slice.sample_rate, fit_range=(1e2, 23e3), num_points_per_block=350)
+
+To fit the passive calibration data, we will use a model based on a number of publications by the Flyvbjerg group :cite:`berg2004power,tolic2004matlab,hansen2006tweezercalib,berg2006power`.
+This model can be found in `PassiveCalibrationModel`. It is calibrated by fitting the following equation to the power spectrum:
```

Comment:
Use the Sphinx notation to link this class to its API reference page.

Reply:
👍